Ejercicio2/src/main/java/com/mycompany/ejercicio2/Ave.java

```diff
@@ -0,0 +1,15 @@
+package com.mycompany.ejercicio2;
+
+public class Ave extends Animal {
+
+    public String trinar(String trino) {
+
+        return trino;
```

Comment:
Please add something else on this function to this return, do not just return the variable itself in this case

For example:

`return "Esto es el trino:" + trino`